rpc_jobs/standard_job_lint_jenkins.yml

```diff
@@ -35,7 +35,7 @@
           trigger-phrase: '.*recheck_all.*|.*recheck_lint_jenkins.*'
           only-trigger-phrase: false
           white-list-target-branches:
-            - "{branches}"
+            - "{branch}"
```

Comment:
The branch param is specified in the docs, for projects using this template: https://rpc-openstack.atlassian.net/wiki/spaces/RE/pages/522223629/Managing+JJB+definitions+across+repositories